packages/synchro-charts/src/utils/dataTypes.ts

```diff
@@ -81,6 +81,15 @@ export interface SizeConfig extends MinimalSizeConfig {
   marginBottom: number;
 }
 
+export interface DialSizeConfig extends MinimalSizeConfig {
+  fontSize: number;
```

Comment:
should document the expected units for each of these somewhere

Reply:
Sorry, I'm not quite sure what you mean. Defining `DialSizeConfig ` is just used to sc-dial-svg  component. The default unit of svg is 'px'. And I guess that I  don't have to set the units in svg.  Or do you have any suggestions for units?